src/setupTests.js

```diff
@@ -21,21 +21,21 @@ Object.defineProperties(window, {
   }
 });
 
-const AllProviders = ({ mocks, initialValue, children }) => {
+const AllProviders = ({ mocks, initialValue, initialPath = '/',children }) => {
```

Comment:
missed a ` ` after `,`. You should run your lint or check why it didn't fix it for you